src/components/NotFoundMessage/NotFoundMessage.tsx

```diff
@@ -0,0 +1,15 @@
+import '~/assets/scss/routes/page-not-found.scss';
+
+export function NotFoundMessage() {
+  return (
+    <>
+      <section class="page-404">

```

Comment:
Currently something like /locations/999999 returns this but with a 200 HTTP code, so maybe we can add a <HttpStatusCode code={404} /> here to ensure when this is used we also get the correct HTTP code.